tests/test_constrained.py

```diff
@@ -2003,3 +2031,6 @@ def test_set_weight_binary(self):
         self.assertFalse(cqm.constraints[c].lhs.is_soft())
         self.assertEqual(cqm.constraints[c].lhs.weight(), float('inf'))
         self.assertIs(cqm.constraints[c].lhs.penalty(), None)
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
This is not needed to run the tests. In fact it's preferred to do `python -m unittest tests/test_constrained.py`